<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_wizard</class>
 <widget class="QDialog" name="window_wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>763</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qressources.qrc">
    <normaloff>:/app/images/icons/icon.png</normaloff>:/app/images/icons/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_icon_title">
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">ICON</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_title">
         <property name="styleSheet">
          <string notr="true">color:rgb(40, 37, 37);</string>
         </property>
         <property name="text">
          <string notr="true">MemberStat</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_widget">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QProgressBar" name="progressBarWelcome">
       <property name="styleSheet">
        <string notr="true">QProgressBar::chunk{
	background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
	stop: 0 #c3083f,
	stop: 0.5 #950741,
	stop: 1 #6f2232 );
}

QProgressBar {
	border: 1px solid black;
   	background-color: black;
	color: white;
}</string>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="format">
        <string>Welcome</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarDatabase">
       <property name="styleSheet">
        <string notr="true">QProgressBar::chunk{
	background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
	stop: 0 #c3083f,
	stop: 0.5 #950741,
	stop: 1 #6f2232 );
}

QProgressBar {
	border: 1px solid black;
   	background-color: black;
	color: white;
}</string>
       </property>
       <property name="value">
        <number>15</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="format">
        <string>Database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarSettings">
       <property name="styleSheet">
        <string notr="true">QProgressBar::chunk{
	background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
	stop: 0 #c3083f,
	stop: 0.5 #950741,
	stop: 1 #6f2232 );
}

QProgressBar {
	border: 1px solid black;
   	background-color: black;
	color: white;
}</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="format">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarFinish">
       <property name="styleSheet">
        <string notr="true">QProgressBar::chunk{
	background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
	stop: 0 #c3083f,
	stop: 0.5 #950741,
	stop: 1 #6f2232 );
}

QProgressBar {
	border: 1px solid black;
   	background-color: black;
	color: white;
}</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="format">
        <string>Finish</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qressources.qrc"/>
 </resources>
 <connections/>
</ui>
